backend/src/main/java/gov/cdc/usds/simplereport/service/model/PatientEmailsHolder.java

```diff
@@ -0,0 +1,42 @@
+package gov.cdc.usds.simplereport.service.model;
+
+import java.util.Collections;
+import java.util.List;
+
+public class PatientEmailsHolder {
```

Comment:
I dont like this file, it doesn't provide any value, we dont have a notion of default email.

Reply:
Touched on in https://github.com/CDCgov/prime-simplereport/pull/2897#discussion_r743999626 but this is _specifically_ a shim to temporarily support backwards compatibility of the GraphQL schema.

When we do the work to close this schema drift this class can be removed and the controllers and resolvers can use the incoming `emails` property directly. In the meantime, though, it is replacing the heinous nested ternary `emails != null ? emails : email == null ? null : List.of(email);` (see 97fbc17).